game/building.cpp

```diff
@@ -1164,13 +1164,20 @@ float Building::getTerrainAnimationMoveSpeed()
     }
 }
 
-bool Building::canRepair(Unit* pUnit)
+bool Building::canRepair(Unit* pUnit, bool always = false)
```

Comment:
This is wrong.
The line in the header building.h should be:
Q_INVOKABLE bool canRepair(Unit* pUnit, bool always = false);
The line in the header building.cpp should be:
bool Building::canRepair(Unit* pUnit, bool always)

I know I can fix after the PR, but I would prefer it the other way around